extensions/wikia/AdEngine/js/AdSlotBottomLeaderboard.run.js

```diff
@@ -11,9 +11,9 @@ require([
 		return false;
 	}
 
-	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slot, provider) {
+	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slotName, provider) {
 
-		if (slot[0].indexOf('LEADERBOARD') !== -1 && (slot[2] === 'Liftium' || provider.name === 'Later')) {
+		if (slotName.indexOf('LEADERBOARD') !== -1 && (provider.name === 'Liftium' || provider.name === 'Later')) {
```

Comment:
This makes sense with when cycled in loops. I believe memory allocation + garbage collecting is more consuming then accessing property in local variable twice, doesn't it?